.editorconfig

```diff
@@ -4,7 +4,6 @@ root = true
 [*]
 charset = utf-8
 end_of_line = lf
-indent_size = 2
-indent_style = space
+indent_style = tab
```

Comment:
This one seems pretty aggressive :)
Tabs vs spaces is a holy war, let's live and code in peace :)

Reply:
Sorry, I went through your arguments. Seems ok to me. Dying to see a good visual programming IDE, that would solve indentation struggle once and for all :)